LegionInvasionTimer.lua

```diff
@@ -186,43 +159,30 @@ frame:SetScript("OnEvent", function(f)
 		GameTooltip:Show()
 	end)
 	f:SetScript("OnLeave", GameTooltip_Hide)
-	local bg = f:CreateTexture(nil, "PARENT")
-	bg:SetAllPoints(f)
-	bg:SetColorTexture(0, 1, 0, 0.3)
-	f.bg = bg
-	local header = f:CreateFontString(nil, "OVERLAY", "TextStatusBarText")
-	header:SetAllPoints(f)
-	header:SetText(name)
-	f.header = header
-	f.startBar = startBar
-	f.db = legionTimerDB
+	f.bg = f:CreateTexture(nil, "PARENT")
```

Comment:
This is an unnecessary change that produces a minuscule performance downgrade. This method forces a table lookup every single time a change to a widget is made. If you are accessing a table entry multiple times, it is more efficient to do so via a local variable.